src/isar/robot/robot_start_mission.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from threading import Event, Thread
+from typing import Optional
+from isar.config.settings import settings
+from isar.models.communication.queues.queues import Queues
+from isar.robot.robot_queue_utils import trigger_event
+from isar.services.utilities.threaded_request import (
+    ThreadedRequest,
+)
+from robot_interface.models.exceptions.robot_exceptions import (
+    ErrorMessage,
+    RobotException,
+)
+from robot_interface.robot_interface import RobotInterface
+
+from robot_interface.models.mission.mission import Mission
+from robot_interface.models.exceptions.robot_exceptions import (
+    RobotInfeasibleMissionException,
+)
```

Comment:
Sort the imports

Reply:
I ran it and it changed many more files than my PR already changes, so I would rather run it in another PR. There is already enough issues with merge conflicts in this PR as-is.